experimental/packages/opentelemetry-instrumentation-http/src/types.ts

```diff
@@ -87,4 +87,6 @@ export interface HttpInstrumentationConfig extends InstrumentationConfig {
     client?: { requestHeaders?: string[]; responseHeaders?: string[] };
     server?: { requestHeaders?: string[]; responseHeaders?: string[] };
   };
+  /** Enable automatic population of synthetic source type based on the user-agent header */
```

Comment:
Please add an `@experimental` annotation on this so that end-users know this might change or go away.